### PR TITLE
Bepis goes along the crate cost changes

### LINKED ISSUE #55525: bepis was forgotten by the crate balance change
all cargo stuff and exports got halved but bepis still uses old values, idk how to tweak the gaussian values to use half money

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request
Makes the bepis scale with the crate value changes
fixes #55525

## Why It's Good For The Game
mmmm bepis

## Changelog
:cl:
fix: The B.E.P.I.S market has been stabilized
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
did you do everything / 2 ? cause it doesnt work, thats the reason i asked the maker to fix the gaussian value

**Comment by spessbro** [comment]
> did you do everything / 2 ? cause it doesnt work, thats the reason i asked the maker to fix the gaussian value

as you can see i just made the biggest values in accordance with the new crate value (i don't actually know exactly how the bepis works)

**Comment by Tlaltecuhtli** [comment]
if it gives 86% on 1025 cr deposited at tier 4 it means it works else its not

**Comment by spessbro** [comment]
fixed but more (I think)

File: code/modules/research/bepis.dm [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 //economics defined values, subject to change should anything be too high or low in practice.
 
-#define MACHINE_OPERATION 100000
-#define MACHINE_OVERLOAD 500000
-#define MAJOR_THRESHOLD 3000
-#define MINOR_THRESHOLD 2000
-#define STANDARD_DEVIATION 1000
+#define MACHINE_OPERATION 400*CARGO_CRATE_VALUE
+#define MACHINE_OVERLOAD 2000*CARGO_CRATE_VALUE
```
Comment (ArcaneMusic) [inline code/modules/research/bepis.dm:6]: These should be unchanged as they're the power cost of the BEPIS, not credit values.

File: code/modules/research/bepis.dm [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
+#define MAJOR_THRESHOLD 12*CARGO_CRATE_VALUE
+#define MINOR_THRESHOLD 8*CARGO_CRATE_VALUE
+#define STANDARD_DEVIATION 4*CARGO_CRATE_VALUE
```
Comment (ArcaneMusic) [inline code/modules/research/bepis.dm:9]: To keep the pricing consistent with how crates were balanced, it should probably be 6x for major, 4x for minor, and 2x for the standard deviation.

Comment (spessbro) [reply code/modules/research/bepis.dm:9]: SO i was right the first time!

**Pull request merged**